src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,56 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    public static final String SUPPLY = "supply";
+    public static final String BUY = "buy";
+    public static final String SEPARATOR = ",";
+    public static final String RESULT = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName));
+                BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(toFileName))) {
+
+            String value = bufferedReader.readLine();
+            int supplyCount = 0;
+            int buyCount = 0;
+
+            while (value != null) {
+                String[] currentLine = value.split(SEPARATOR);
+                if (currentLine.length == 2) {
```

Comment:
The use of `split()` here is correct as per the checklist. However, consider adding error handling for cases where the line format is not as expected (e.g., not enough elements after split).